python/lsst/cp/pipe/linearity.py

```diff
@@ -453,6 +453,15 @@ def run(self, inputPtc, dummy, camera, inputDims, inputPhotodiodeData=None,
                     binCenters = binCenters[numPerBin > 0]
 
                 self.debugFit('splineFit', binCenters, np.abs(values), values, None, ampName)
+                # Anchor the spline to have zero correction at zero
+                # flux as well as at the lowest measured flux bin.
+                if binCenters[0] != 0.0:
+                    if values[0] != 0.0:
```

Comment:
Does this behave appropriately in both cases of `values[0]<0` and `values[0]>0`? Just wondering if you can end up with negative `values` and if that's worth checking or if that's only possible if you did something obviously wrong in the first place.

Reply:
Negative values in the correction is fine; the `values` correspond to amounts to be added to pixels at that flux, and negative values simply means some amount needs to be subtracted.